app/src/scss/_utility-classes.scss

```diff
@@ -19,3 +19,168 @@
   width: 1px !important;
   white-space: nowrap !important;
 }
+
+/* TYPOGRAPHY */
+// TODO: clean up these util classes
+// big header / title fonts on the home page
+.font-style-hero {
```

Comment:
wondering if at least removing all the `width` variations might be possible now that `text-wrap: balance` is being used?